### DISCUSSION TITLE
Will I be able to get a GPT-4 API key w/ this? (ChatGPT Plus)

### DISCUSSION THREAD

**Discussion by Arche151** [acheong08/ChatGPT#1253] [Q&A]
I am still kind of confused about what I can do with this repo. I am still waiting for access to the official GPT-4 API, but I am able to use GPT-4 via my ChatGPT-Plus account. Does this repo make it possible for me to get a GPT-4 API key by using my ChatGPT-Plus account, and if yes, how exactly?

Thank you in advance!

**Comment by acheong08** [comment]
You can use GPT-4 with V1 via your plus account by setting the model to gpt-4 in config.

**Reply by Arche151** [reply]
Cool, thanks! Do you know if there's a risk of getting banned?
